docs/source/what-is.rst

```diff
@@ -22,11 +22,25 @@ in the ML lifecycle.
    :alt: An architecture diagram of SageMaker JumpStart end-to-end solutions
        and the coverage of SageMaker JumpStart Industry Python SDK
 
-#. Use SageMaker JumpStart Industry notebooks for solutions, models, and examples.
+#. Use `SageMaker JumpStart Industry
+   <https://docs.aws.amazon.com/sagemaker/latest/dg/studio-jumpstart-industry.html>`_
+   notebooks for solutions, models, and examples.
```

Comment:
the notebooks in the repo is not used in the solution

Reply:
Yes, I understand that. Here, I generally spoke about the entire smjsindustry notebooks.

Reply:
I am not talking that the notebook in the repo is what's in the solution. I am gererally saying that the solutions, models, and example notebooks are available to use.

Reply:
https://sagemaker-jumpstart-industry-pack--24.org.readthedocs.build/en/24/what-is.html#how-it-works

<img width="765" alt="Screen Shot 2021-09-29 at 5 30 34 PM" src="https://user-images.githubusercontent.com/21208346/135366472-b4e133b6-9e96-4f5d-b6e6-6bb5002a1e68.png">

Reply:
@sophiayue1116 Rephrased it: https://sagemaker-jumpstart-industry-pack--24.org.readthedocs.build/en/24/what-is.html#how-it-works